spec/services/event_finder_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe EventFinder do
+  describe ".next_event" do
+    it "returns the first upcoming event" do
+      group_name = "lightening-talks"
+
+      events = build_events(group_name)
+
+      allow(EventService)
+        .to receive(:upcoming_events)
+        .with(group_name: group_name)
+        .and_return(events)
```

Comment:
Have you considered extracting this out to a helper method that accepts the group name and return value as arguments?